libft/sources/map/mapss/ft_mapss_insert.c

```diff
@@ -0,0 +1,99 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_mapss_insert.c                                  :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: taretiuk <taretiuk@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/08/09 13:20:33 by taretiuk          #+#    #+#             */
+/*   Updated: 2024/08/14 21:06:42 by inikulin         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../../../libft.h"
+#include <stdlib.h>
+
+int	ft_mapss_insert(t_mapss *map, t_dlist *node)
```

Comment:
TODO: please split into separate functions inside this very file - to meet the Norm